### ISSUE TITLE
Create Website Documentation Specifically Aimed at Developers

### ISSUE THREAD

**Issue by jamescummings** [TEIC/TEI#1873]
TEI-C website should have webpage aimed specifically at briefly explaining a summary of the TEI and some bits of its infrastructure that may be useful in building TEI-related software. 

For example, MarjorieB asked me recently how an XML editor plugin can easily know that there is a new version of the TEI released. And simply it is check to see if this file: 
https://www.tei-c.org/Vault/P5/current/doc/tei-p5-doc/VERSION
has a different number in it. But it took me awhile (about 5 minutes) to think of that as a solution, even though in hindsight it is an obvious way to check.  So "How to programmatically check if there is a new TEI schema version available" is the kind of developer FAQ that might be answered on such a page. I'm sure we can all think of other things that should be there.

**Comment by MarjorieBurghart** [comment]
Yes, it would be much appreciated. Of course a web service would be a cleaner solution, but just publicizing the existence of this plain text file would be great.

**Comment by ebeshero** [comment]
Council F2F: We should start working on this in the TEI wiki. We'll start by assigning a Council member to create the space on the TEI wiki and invite people to start working on this.

**Comment by jamescummings** [comment]
@peterstadler I started a wiki page here https://wiki.tei-c.org/index.php?title=FAQ-Developers where we can start gathering questions/answers when they occur to us. Eventually we could have a page on the tei-c.org site point there.
